modules/end-user-guide/pages/url-parameter-for-workspace-image.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: CONCEPT
+:description: URL parameter for the workspace image
+:keywords: parameter, URL, workspace, image
+:navtitle: URL parameter for the workspace image
+//:page-aliases: url-parameter-for-workspace-image.adoc
+
+[id="url-parameter-for-workspace-image"]
+= URL parameter for the workspace image
```

Comment:
I'd be cautious about using the phrase `workspace image`, which is also in the file name. This topic seems to be about additional containers and their images, so to me this looks like `additional container image` rather than `workspace image`. I might be wrong though, I'm sure the developer can clarify it.

Reply:
Greatpoint, I mashed together workspace and container image without thinking too much about it. The original RN text uses container image.